scalikejdbc-interpolation/src/test/scala/scalikejdbc/QueryInterfaceSpec.scala

```diff
@@ -353,7 +353,7 @@ class QueryInterfaceSpec extends FlatSpec with Matchers with DBSettings with SQL
         {
           val notInClauseResults = withSQL {
             select.from(Order as o)
-              .where.not.in(o.id, Seq())
+              .where.not.in(o.id, Seq[Int]())
```

Comment:
In general, it's ok for us, well-trained Scala programmers, but OTOH, it may be a bit confusing error for Scala beginners. Is there any way to improve the error message to suggest what to do?

```scala
[error] src/test/scala/scalikejdbc/QueryInterfaceSpec.scala:333: ambiguous implicit values:
[error]  both value sqlArrayParameterBinderFactory in object ParameterBinderFactory of type => scalikejdbc.ParameterBinderFactory[java.sql.Array]
[error]  and value bigDecimalParameterBinderFactory in object ParameterBinderFactory of type => scalikejdbc.ParameterBinderFactory[BigDecimal]
[error]  match expected type scalikejdbc.ParameterBinderFactory[B]
[error]               .where.in(o.id, Seq.empty)
[error]                        ^
```